util/cache/none.go

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package cache
+
+// None provides no caching and always calls the provided FillFunc.
+//
+// It is safe for concurrent use.
```

Comment:
Does this depend technically on whether the FillFunc is?

Reply:
Yes, good point; will update the comment 👍